cmake_autoformat.bat

```diff
@@ -0,0 +1,22 @@
+@echo off
```

Comment:
Could we run `cmake-format` using the [`pre-commit`](https://pre-commit.com/) tool rather than adding our own custom scripts? There is a `pre-commit` hook here we could use: https://github.com/cheshirekow/cmake-format-precommit. We get a lot of useful stuff out of the box with `pre-commit`. It will also allow us to add more formatting and code style checks easily in the future without the requiring us to add multiple "runner" scripts alongside them.

Reply:
This would be a good start to start running astyle checks as well 👍